### DISCUSSION TITLE
Query on available resources for validating  ML workloads

### DISCUSSION THREAD

**Discussion by BA-ahmed** [rocm-arch/rocm-arch#788] [Support] [answered]
I am trying to understand the behavior of AI/ML workload (such as DNN training by PyTorch and TensorFlow) on AMD targets like Rembrandt generation APU (Ryzen 6000 series).

I would like to know the tests available that can be used to validate regressions and also validate what features are working well.
Can anyone help me answer this?

**Comment by acxz** [comment, marked as answer]
This is a pretty good question. Sadly, we do not have tests and checks builtin the PKGBUILDs as we have primarily been focusing on just getting the stack to build. However, now that most of the stack builds properly, maybe we should look into adding a `check()` function to our PKGBUILDS.

To answer your specific question on the AI/ML workflow, we are still working through the `python-pytorch-rocm` and `tensorflow-rocm` packages at the corresponding repos at https://github.com/rocm-arch.

Another thing is that you can compile those ML libraries from source yourself and run the tests for those packages located on their repos.
